examples/index.css

```diff
@@ -1,3 +1,7 @@
+@import "~normalize.css";
```

Comment:
normally `@import` imports a local file. When you prefix it with `~`, it loads the file from a node module.
So basically it's doing `@import(${require.resolve('normalize.css')})`. `normalize.css` is a node module.